backend/substrapp/tasks/tasks.py

```diff
@@ -462,6 +471,11 @@ def compute_task(self, tuple_type, subtuple, compute_plan_id):
         error_code = compute_error_code(e)
         logging.error(error_code, exc_info=True)
 
+        # Trigger the celery auto-retry if this isn't our last attempt
```

Comment:
I think we should just let the exception raise here? without catching? otherwise celery doc recommand to use self.retry 
https://docs.celeryproject.org/en/latest/userguide/tasks.html#using-a-custom-retry-delay

Reply:
Fixed https://github.com/SubstraFoundation/substra-backend/pull/66/commits/0462f788d66e64ff1ccc02b9d751ac377b434ea2